templates/repo/settings/webhook/gitea.tmpl

```diff
@@ -1,40 +1,40 @@
-		{{.CsrfTokenHtml}}
-		<div class="required field {{if .Err_PayloadURL}}error{{end}}">
-			<label for="payload_url">{{.i18n.Tr "repo.settings.payload_url"}}</label>
-			<input id="payload_url" name="payload_url" type="url" value="{{.Webhook.URL}}" autofocus required>
-		</div>
-		<div class="field">
-			<label>{{.i18n.Tr "repo.settings.http_method"}}</label>
-			<div class="ui selection dropdown">
-				<input type="hidden" id="http_method" name="http_method" value="{{if .Webhook.HTTPMethod}}{{.Webhook.HTTPMethod}}{{else}}POST{{end}}">
-				<div class="default text"></div>
-				<i class="dropdown icon"></i>
-				<div class="menu">
-					<div class="item" data-value="POST">POST</div>
-					<div class="item" data-value="GET">GET</div>
-				</div>
-			</div>
-		</div>
-		<div class="field">
-			<label>{{.i18n.Tr "repo.settings.content_type"}}</label>
-			<div class="ui selection dropdown">
-				<input type="hidden" id="content_type" name="content_type" value="{{if .Webhook.ContentType}}{{.Webhook.ContentType}}{{else}}application/json{{end}}">
-				<div class="default text"></div>
-				<i class="dropdown icon"></i>
-				<div class="menu">
-					<div class="item" data-value="1">application/json</div>
-					<div class="item" data-value="2">application/x-www-form-urlencoded</div>
-				</div>
-			</div>
-		</div>
-		<input class="fake" type="password">
-		<div class="field {{if .Err_Secret}}error{{end}}">
-			<label for="secret">{{.i18n.Tr "repo.settings.secret"}}</label>
-			<input id="secret" name="secret" type="password" value="{{.Webhook.Secret}}" autocomplete="off">
-		</div>
-		{{template "repo/settings/webhook/settings" .}}
-	</form>
-{{end}}
+{{if eq .HookType "gitea"}}
+	<p>{{.i18n.Tr "repo.settings.add_webhook_desc" "https://docs.gitea.io/en-us/webhooks/" | Str2html}}</p>
+	<form class="ui form" action="{{.BaseLink}}/gitea/{{or .Webhook.ID "new"}}" method="post">
+		{{.CsrfTokenHtml}}
+		<div class="required field {{if .Err_PayloadURL}}error{{end}}">
+			<label for="payload_url">{{.i18n.Tr "repo.settings.payload_url"}}</label>
+			<input id="payload_url" name="payload_url" type="url" value="{{.Webhook.URL}}" autofocus required>
+		</div>
+		<div class="field">
+			<label>{{.i18n.Tr "repo.settings.http_method"}}</label>
+			<div class="ui selection dropdown">
+				<input type="hidden" id="http_method" name="http_method" value="{{if .Webhook.HTTPMethod}}{{.Webhook.HTTPMethod}}{{else}}POST{{end}}">
+				<div class="default text"></div>
+				<i class="dropdown icon"></i>
+				<div class="menu">
+					<div class="item" data-value="POST">POST</div>
+					<div class="item" data-value="GET">GET</div>
+				</div>
+			</div>
+		</div>
+		<div class="field">
+			<label>{{.i18n.Tr "repo.settings.content_type"}}</label>
+			<div class="ui selection dropdown">
+				<input type="hidden" id="content_type" name="content_type" value="{{if .Webhook.ContentType}}{{.Webhook.ContentType}}{{else}}1{{end}}">
```

Comment:
why change ContentType for gitea too?

Reply:
The same problem occurs with the Gitea webhook if you don't select a ContentType from the drop down. The intention of the template seemed to be to populate a working value in the form as a default, but that's not what's happening.

![screenshot-20200225-142005](https://user-images.githubusercontent.com/293465/75255675-29221880-5797-11ea-96e5-a18c7e64e047.png)